packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -136,6 +149,12 @@ export default function OrgNav({
     }
   }
 
+  const paymentValues: Array<OrgPaymentLabel> = [
+    'SUSPENDED',
```

Comment:
Values should still be enum values, `OrgPaymentLabel.Suspended` etc.

Reply:
Fixed!